Tribler/Core/defaults.py

```diff
@@ -251,6 +252,7 @@
 tribler_defaults['Tribler']['window_height'] = 670
 tribler_defaults['Tribler']['sash_position'] = -185
 tribler_defaults['Tribler']['family_filter'] = 1
+tribler_defaults['Tribler']['monochrome_tray_icon'] = False
```

Comment:
Looks good! One more thing: maybe you could refactor the code that sets the tray icon in a separate method (named like `update_tray_icon`) and call it when the newly added checkbox state changes. This way, the update of the tray icon happens immediately and no restart is needed. Can you implement this? 👍 

When done, I will merge your PR.